Skyra.Tests/Core/Utils/EmojiRegexTests.cs

```diff
@@ -0,0 +1,38 @@
+using JetBrains.Annotations;
+using NUnit.Framework;
+using Skyra.Core.Utils;
+
+namespace Skyra.Tests.Core.Utils
+{
+	public class EmojiRegexTests
+	{
+		[Test]
```

Comment:
Adding `[Parallelizable]` to both tests will allow the test to run in... well, parallel (who'd have thought?)